deno.lock

```diff
@@ -1,75 +1,88 @@
 {
```

Comment:
My previous PR accidentally removed all test dependencies from the lock file :sweat: